packages/native-stack/src/utils/getModalRoutesKeys.ts

```diff
@@ -0,0 +1,21 @@
+import type { Route } from '@react-navigation/native';
+
+import type { NativeStackDescriptorMap } from '../types';
+
+export const getModalRouteKeys = (
```

Comment:
@satya164 can we place this util somewhere to share between the packages?
cause now I made a dublicate for native and simple stack :(

Reply:
It's fine to duplicate. There's not really much logic here.